app/carbonapi/api.go

```diff
@@ -40,26 +41,17 @@ import (
 // BuildVersion is provided to be overridden at build time. Eg. go build -ldflags -X 'main.BuildVersion=...'
 var BuildVersion string
 
-var config = struct {
-	cfg.API
```

Comment:
Was there a problem with embedding the `cfg.API` struct?

Reply:
The EnvConfig struct embeds the cfg.API struct and some other configs as well, similar to the previous configs for api and zipper. The zipper EnvConfig wraps the cfg.Zipper and list of backends.

Reply:
@gunnihinn do you have any concerns regarding this?